gulp/tasks/copy.js

```diff
@@ -22,6 +22,11 @@ gulp.task( 'copy:icons', () => {
   return _genericCopy( icons.src, icons.dest );
 } );
 
+gulp.task( 'copy:iconsOld', () => {
```

Comment:
Could use `TODOs` above these `iconsOld` code bits about when to remove them since I presume they're only intended to be around for a sprint or so?

Reply:
Hey, I'll work on this while Jimmy's out today. If I add todos, I get linter warnings for this rule: https://eslint.org/docs/rules/no-warning-comments

We apparently just let these warnings go everywhere. How about we just change our `.eslintrc` to not warn on them?

Reply:
That's kind of by design so that TODOs nag us, if the code is fixed and the TODO is removed the warning disappears. You can run `gulp lint --quiet` to not look at the warnings.

Reply:
This'll be more than a few sprints. We need to merge and release the animated icon and remove the webfont entirely from CF before it'll go away here. Based on how long it's taken just to finalize this work I would expect more like a month or two.